bpo-45155 : Default arguments for int.to_bytes(length=1, byteorder=sys.byteorder) (#28265)
python/cpython: Doc/library/stdtypes.rst

        >>> x.to_bytes((x.bit_length() + 7) // 8, byteorder='little')
        b'\xe8\x03'

    The integer is represented using *length* bytes, and defaults to 1.  An
    :exc:`OverflowError` is raised if the integer is not representable with
    the given number of bytes.

    The *byteorder* argument determines the byte order used to represent the

>     integer, and defaults to :data:`sys.byteorder`.  If *byteorder* is

Reviewer: ```suggestion
    integer, and defaults to ``"big"``.  If *byteorder* is
```

    ``"big"``, the most significant byte is at the beginning of the byte
    array.  If *byteorder* is ``"little"``, the most significant byte is at
    the end of the byte array.

    The *signed* argument determines whether two's complement is used to
    represent the integer.  If *signed* is ``False`` and a negative integer is
    given, an :exc:`OverflowError` is raised. The default value for *signed*
    is ``False``.

    The default values can be used to conveniently turn an integer into a
    single byte object.  However, when using the default arguments, don't try

>     to convert a value greater than 255 or you'll get an :exc:`OverflowError`.::

Reviewer: This will currently display with both a period and a colon. I'm guessing you only want the colon:

```suggestion
    to convert a value greater than 255 or you'll get an :exc:`OverflowError`::
```

        >>> (65).to_bytes()
        b'A'

    Equivalent to::

>         def to_bytes(n, length=1, byteorder=sys.byteorder, signed=False):

Reviewer: ```suggestion
        def to_bytes(n, length=1, byteorder='big', signed=False):
```

        >>> x.to_bytes((x.bit_length() + 7) // 8, byteorder='little')
        b'\xe8\x03'

    The integer is represented using *length* bytes, and defaults to 1.  An
    :exc:`OverflowError` is raised if the integer is not representable with
    the given number of bytes.

    The *byteorder* argument determines the byte order used to represent the
    integer, and defaults to :data:`sys.byteorder`.  If *byteorder* is
    ``"big"``, the most significant byte is at the beginning of the byte
    array.  If *byteorder* is ``"little"``, the most significant byte is at
    the end of the byte array.  To request the native byte order of the host

>     system, use :data:`sys.byteorder` as the byte order value.

Reviewer 2: The last sentence can just be removed now, because that's the default.

    iterable producing bytes.

    The *byteorder* argument determines the byte order used to represent the

>     integer, and defaults to :data:`sys.byteorder`.  If *byteorder* is

Reviewer: ```suggestion
    integer, and defaults to ``"big"``.  If *byteorder* is
```

    ``"big"``, the most significant byte is at the beginning of the byte
    array.  If *byteorder* is ``"little"``, the most significant byte is at
    the end of the byte array.  To request the native byte order of the host
    system, use :data:`sys.byteorder` as the byte order value.

    The *signed* argument indicates whether two's complement is used to
    represent the integer.

    Equivalent to::

>         def from_bytes(bytes, byteorder=sys.byteorder, signed=False):

Reviewer: ```suggestion
        def from_bytes(bytes, byteorder='big', signed=False):
```
